sample.py

```
@@ -0,0 +1,26 @@
+
+import *    
+import os,sys;import math     
+
+A=1
+A=1
+def DoSomething(x,y):return x+y+ A 
+
+class myclass:
+	def __init__(self):self.Data=[1,"two",3.0];self.v=42 
```

Ruff (0.14.8)

Multiple statements on one line (semicolon)

(E702)